.eslintrc

```diff
@@ -0,0 +1,38 @@
+{
+  "env": {
+    "node": true
+  },
+  "globals": {},
+  "rules": {
+    "no-bitwise": 2,
+    "camelcase": 0,
+    "curly": 2,
+    "eqeqeq": 2,
+    "no-unused-expressions": [
+      2,
+      "allowTernary": true
```

Comment:
Is this kosher, or should it be an object? http://jsonlint.com/ fails pretty hard here.

```js
    "no-unused-expressions": [
      2,
      {
        "allowTernary": true
      }
    ],
```